mobile/App.tsx

```diff
@@ -15,19 +17,24 @@ export default function App() {
         <Stack.Screen
           name="Home"
           component={Authentication}
-          options={{headerShown: false}}
+          options={{ headerShown: false }}
         />
-        <Stack.Screen 
+        <Stack.Screen
           name="Login"
           component={Login}
-          options={{headerShown: false}}
+          options={{ headerShown: false }}
         />
-        <Stack.Screen 
+        <Stack.Screen
           name="Register"
           component={Register}
-          options={{headerShown: false}}
+          options={{ headerShown: false }}
+        />
+        <Stack.Screen
```

Comment:
this page will eventually be refactored with a auth flow in mind [[](https://reactnavigation.org/docs/auth-flow)](https://reactnavigation.org/docs/auth-flow)